src/commands/General/Chat Bot Info/help.ts

```diff
@@ -41,10 +41,10 @@ function sortCommandsAlphabetically(_: Command[], __: Command[], firstCategory:
 	flagSupport: true
 })
 export default class extends SkyraCommand {
-	public async init() {
+	public async onLoad() {
 		this.createCustomResolver('command', (arg, possible, message) => {
```

Comment:
decorators?

Reply:
The `category` resolver requires access to the `_fetchCommands` method from that class.